packages/beacon-node/src/network/processor/gossipHandlers.ts

```diff
@@ -281,7 +285,7 @@ function getDefaultHandlers(modules: ValidatorFnsModules, options: GossipHandler
               const forkTypes = config.getForkTypes(slot);
               const rootHex = toHexString(forkTypes.BeaconBlock.hashTreeRoot(signedBlock.message));
 
-              events.emit(NetworkEvent.unknownBlock, {rootHex, peer: peerIdStr});
+              events.emit(NetworkEvent.unknownBlock, {blockInputOrRootHex: rootHex, peer: peerIdStr});
 
               // Error is quite frequent and not critical
               logLevel = LogLevel.debug;
```

Comment:
to prevent any races, anyway it doesn't hurt and possibly can be removed after some rigorous testing, but right now emphasis is to make sure we can aggressive pull the block and blobs